packages/react-native-editor/__device-tests__/gutenberg-editor-slash-inserter-@canary.test.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Internal dependencies
+ */
+import { blockNames } from './pages/editor-page';
+import { isAndroid } from './helpers/utils';
+import { slashInserter, shortText } from './helpers/test-data';
+
+const ANIMATION_TIME = 200;
+
+describe( 'Gutenberg Editor Slash Inserter tests', () => {
+	it( 'should show the menu after typing /', async () => {
+		await editorPage.addNewBlock( blockNames.paragraph );
+		const paragraphBlockElement = await editorPage.getBlockAtPosition(
+			blockNames.paragraph
+		);
+		if ( isAndroid() ) {
+			await paragraphBlockElement.click();
+		}
+
+		await editorPage.typeTextToParagraphBlock(
+			paragraphBlockElement,
+			slashInserter
+		);
+		await editorPage.driver.sleep( ANIMATION_TIME );
+
+		// Slash inserter
+		const slashInserterElement = await editorPage.driver.elementByAccessibilityId(
+			'Slash inserter results'
+		);
+		expect( slashInserterElement ).toBeTruthy();
+		await editorPage.removeBlockAtPosition( blockNames.paragraph );
+	} );
+
+	it( 'should hide the menu after deleting the / character', async () => {
+		await editorPage.addNewBlock( blockNames.paragraph );
+		const paragraphBlockElement = await editorPage.getBlockAtPosition(
+			blockNames.paragraph
+		);
+		if ( isAndroid() ) {
+			await paragraphBlockElement.click();
+		}
+
+		await editorPage.typeTextToParagraphBlock(
+			paragraphBlockElement,
+			slashInserter
+		);
+		await editorPage.driver.sleep( ANIMATION_TIME );
+
+		// Slash inserter
+		const slashInserterElement = await editorPage.driver.elementByAccessibilityId(
+			'Slash inserter results'
+		);
+		expect( slashInserterElement ).toBeTruthy();
+
+		// Remove / character
+		if ( isAndroid() ) {
+			await editorPage.typeTextToParagraphBlock(
+				paragraphBlockElement,
+				`${ shortText }`,
+				true
+			);
+		} else {
+			await editorPage.typeTextToParagraphBlock(
+				paragraphBlockElement,
+				`\b ${ shortText }`,
+				false
+			);
+		}
+		await editorPage.driver.sleep( ANIMATION_TIME );
+
+		// Check if the slash inserter UI exists
+		const foundElements = await editorPage.driver.elementsByAccessibilityId(
+			'Slash inserter results'
+		);
+		expect( foundElements.length ).toBe( 0 );
+
+		await editorPage.removeBlockAtPosition( blockNames.paragraph );
+	} );
+
+	it( 'should add an Image block after tying /image and tapping on the Image block button', async () => {
+		await editorPage.addNewBlock( blockNames.paragraph );
+		const paragraphBlockElement = await editorPage.getBlockAtPosition(
+			blockNames.paragraph
+		);
+		if ( isAndroid() ) {
+			await paragraphBlockElement.click();
+		}
+
+		await editorPage.typeTextToParagraphBlock(
+			paragraphBlockElement,
+			`${ slashInserter }image`
+		);
+		await editorPage.driver.sleep( ANIMATION_TIME );
+
+		// Slash inserter
+		const slashInserterElement = await editorPage.driver.elementByAccessibilityId(
```

Comment:
This code repeats itself a few times, do you think it would be helpful if we create a function for it? 

We could have a `assertSlashInserterPresent` and `assertSlashInserterNotPresent` methods for example.